CHANGELOG/CHANGELOG-2.2.md

```diff
@@ -0,0 +1,70 @@
+  -  `antctl get bgppolicy` to get the effective BGP policy applied on the local Node. ([#6646](https://github.com/antrea-io/antrea/pull/6646), [@Atish-iaf])
+  -  `antctl get bgppeers` to print the current status of all BGP peers of the effective BGPPolicy applied on the local Node. ([#6689](https://github.com/antrea-io/antrea/pull/6689) [#6755](https://github.com/antrea-io/antrea/pull/6755), [@Atish-iaf])
+  -  `antctl get bgproutes` to print the advertised BGP routes on the local Node. ([#6734](https://github.com/antrea-io/antrea/pull/6734), [@Atish-iaf])
+- Add an `except` field to the Antrea-native policy field `ipBlock` to allow users to exclude certain CIDRs from `ipBlock.cidr`. ([#6658](https://github.com/antrea-io/antrea/pull/6658) [#6677](https://github.com/antrea-io/antrea/pull/6677), [@Dyanngg])
+- Add a new `templateRefreshTimeout` configuration for `FlowAggregator` to define the template retransmission interval when using the UDP protocol to export records. ([#6699](https://github.com/antrea-io/antrea/pull/6699), [@antoninbas])
+- Add `EnableLogging` and `LogLabel` support for Antrea Node NetworkPolicy. ([#6626](https://github.com/antrea-io/antrea/pull/6626), [@hongliangl])
+- Add `ServiceTrafficDistribution` feature in Antrea Proxy that enables traffic distribution for Services. ([#6604](https://github.com/antrea-io/antrea/pull/6604), [@hongliangl])
+- Add `antctl-darwin-arm64` to Antrea release assets. ([#6640](https://github.com/antrea-io/antrea/pull/6640), [@antoninbas])
+- Add documentation for the `NodeLatencyMonitor` feature. ([#6561](https://github.com/antrea-io/antrea/pull/6561), [@antoninbas])
+
+### Changed
+
+- Uniform BGP router ID selection for IPv4 and IPv6 for the `BGPPolicy` feature. ([#6605](https://github.com/antrea-io/antrea/pull/6605), [@Atish-iaf])
+- Use the default protocol / port when the destination is a Service in Traceflow. ([#6601](https://github.com/antrea-io/antrea/pull/6601), [@Atish-iaf])
+- Add validations for Antrea Node NetworkPolicy to fail invalid configurations. ([#6613](https://github.com/antrea-io/antrea/pull/6613), [@Atish-iaf])
+- More robust system Tier creation / update for Antrea-native policies. ([#6696](https://github.com/antrea-io/antrea/pull/6696), [@antoninbas])
+- Handle `ExternalIPPool` range changes in Egress controller. ([#6685](https://github.com/antrea-io/antrea/pull/6685), [@antoninbas])
+- Close connection to IPFIX collector explicitly on Stop for `FlowAggregator`. ([#6635](https://github.com/antrea-io/antrea/pull/6635), [@antoninbas])
+- Support `--random-fully` for iptables SNAT / MASQUERADE rules. ([#6602](https://github.com/antrea-io/antrea/pull/6602), [@antoninbas])
+- Unify the checker image and make it configurable when running `antctl check cluster`. ([#6579](https://github.com/antrea-io/antrea/pull/6579), [@tnqn])
+- Update the `Finalizer` of `ResourceExport` to be domain-qualified strings. ([#6742](https://github.com/antrea-io/antrea/pull/6742), [@Dyanngg])
```

Comment:
> to be a domain-qualified string

I think there is only one Finalizer in that case?

Reply:
Yes, updated.